### PR TITLE
Updes the deletion warning message

### PR DESCRIPTION
**Pull request by ghatighorias**

### CODE REVIEW

File: web/templates/offered_course/index.html.eex [hunk 1]
```diff
@@ -63,12 +63,15 @@
                 <%= link "Edit", to: offered_course_path(@conn, :edit, offered_course) %>
               </li>
               <li class="mdl-menu__item">
-                <%= link "Delete", to: offered_course_path(@conn, :delete, offered_course), method: :delete, data: [confirm: "Are you sure?"] %>
+                <%= link "Delete", to: offered_course_path(@conn, :delete, offered_course), method: :delete,
+                data: [confirm: """
+                                              Are you sure?
+                By removing an offered course all of its classes and releavant attendances will be removed.
```
Comment (ggpasqualino) [inline web/templates/offered_course/index.html.eex]: releavant -> relevant

File: web/templates/course/index.html.eex [hunk 2]
```diff
@@ -51,11 +51,17 @@
                 <%= link "Edit", to: course_path(@conn, :edit, course) %>
               </li>
               <li class="mdl-menu__item">
-                <%= link "Delete", to: course_path(@conn, :delete, course), method: :delete, data: [confirm: "Are you sure?"] %>
+                <%= link "Delete", to: course_path(@conn, :delete, course), method: :delete,
+                data: [confirm: """
+                                              Are you sure?
+                By removing a course all the offered courses in all terms will be removed,
+                consequently removal of an offered course will follow by removal of all of
+                its classes and releavant attendances.
```
Comment (ggpasqualino) [inline web/templates/course/index.html.eex]: releavant -> relevant

File: web/templates/term/index.html.eex [hunk 3]
```diff
@@ -55,7 +55,12 @@
                 <%= link "Edit", to: term_path(@conn, :edit, term) %>
               </li>
               <li class="mdl-menu__item">
-                <%= link "Delete", to: term_path(@conn, :delete, term), method: :delete, data: [confirm: "Are you sure?"] %>
+                <%= link "Delete", to: term_path(@conn, :delete, term), method: :delete,
+                data: [confirm: """
+                                              Are you sure?
+                By removing a term all of its offered courses and their classes and releavant attendances
```
Comment (ggpasqualino) [inline web/templates/term/index.html.eex]: releavant -> relevant

File: web/templates/student/index.html.eex [hunk 4]
```diff
@@ -55,12 +55,15 @@
                 <%= link "Edit", to: student_path(@conn, :edit, student) %>
               </li>
               <li class="mdl-menu__item">
-                <%= link "Delete", to: student_path(@conn, :delete, student), method: :delete, data: [confirm: "Are you sure?"] %>
+                <%= link "Delete", to: student_path(@conn, :delete, student), method: :delete,
+                data: [confirm: """
+                                              Are you sure?
+                By removing an student all of its data including her attendance history will be removed.
```
Comment (ggpasqualino) [inline web/templates/student/index.html.eex]: an student -> a student

File: web/templates/teacher/index.html.eex [hunk 5]
```diff
@@ -55,11 +55,15 @@
                 <%= link "Edit", to: teacher_path(@conn, :edit, teacher) %>
               </li>
               <li class="mdl-menu__item">
-                <%= link "Delete", to: teacher_path(@conn, :delete, teacher), method: :delete, data: [confirm: "Are you sure?"] %>
+                <%= link "Delete", to: teacher_path(@conn, :delete, teacher), method: :delete,
+                data: [confirm: """
+                                              Are you sure?
+                By removing an teacher the offered course tought by her will be unassigned from her.
```
Comment (ggpasqualino) [inline web/templates/teacher/index.html.eex]: an teacher -> a teacher

Comment (ggpasqualino) [inline web/templates/teacher/index.html.eex]: tought -> taught

File: web/templates/offered_course/index.html.eex (hunk 1 above)
Comment (ghatighorias) [inline web/templates/offered_course/index.html.eex]: done

File: web/templates/course/index.html.eex (hunk 2 above)
Comment (ghatighorias) [inline web/templates/course/index.html.eex]: done

File: web/templates/term/index.html.eex (hunk 3 above)
Comment (ghatighorias) [inline web/templates/term/index.html.eex]: done

File: web/templates/student/index.html.eex (hunk 4 above)
Comment (ghatighorias) [inline web/templates/student/index.html.eex]: done

File: web/templates/teacher/index.html.eex (hunk 5 above)
Comment (ghatighorias) [inline web/templates/teacher/index.html.eex]: both are done

**Pull request merged**